src/test/jekyll/_config_permalink.yaml

```diff
@@ -0,0 +1,85 @@
+# Default jekyll configuration with additional collection for arbitrary path permalink testing
+
+# Where things are
+source: .
+destination: ./_site
+collections_dir: .
+plugins_dir: _plugins # takes an array of strings and loads plugins in that order
+layouts_dir: _layouts
+data_dir: _data
+includes_dir: _includes
+sass:
+  sass_dir: _sass
+collections:
+  posts:
+    output: true
+  my_collection:
+    output: false
+    permalink: /work/:name
+
+# Handling Reading
+safe: false
+include: ['.htaccess']
+exclude:
+  [
+    'Gemfile',
+    'Gemfile.lock',
+    'node_modules',
+    'vendor/bundle/',
+    'vendor/cache/',
+    'vendor/gems/',
+    'vendor/ruby/',
+  ]
+keep_files: ['.git', '.svn']
+encoding: 'utf-8'
+markdown_ext: 'markdown,mkdown,mkdn,mkd,md'
+strict_front_matter: false
+
+# Filtering Content
+show_drafts: null
+limit_posts: 0
+future: false
+unpublished: false
+
+# Plugins
+whitelist: []
+plugins: []
+
+# Conversion
+markdown: kramdown
+highlighter: rouge
+lsi: false
+excerpt_separator: "\n\n"
+incremental: false
+
+# Serving
+detach: false
+port: 4000
+host: 127.0.0.1
+baseurl: '' # does not include hostname
+show_dir_listing: false
+
+# Outputting
+permalink: date
```

Comment:
Usually, test files are as simple as they can (when ensuring there are no regressions for a particular change). Do you think you can remove the as much of the unrelated yaml as you can?